packages/js/passkeys-sdk/README.md

```diff
@@ -0,0 +1,54 @@
+# Hanko Passkeys JavaScript SDK
+
+[View docs on docs.hanko.io](https://docs.hanko.io/passkey-api/js-sdk)
+
+The `@teamhanko/passkeys-sdk` package lets you call the Hanko Passkey API from JavaScript/TypeScript that supports `fetch`.
+
+This includes Node.js, Browsers, Deno, Bun, and frameworks like Next.js, Astro, and SvelteKit.
+
+<CodeGroup>
+
+```bash npm
+npm i @teamhanko/passkeys-sdk
+```
+
+```bash yarn
+yarn add @teamhanko/passkeys-sdk
+```
+
+```bash bun
+bun add @teamhanko/passkeys-sdk
+```
+
+```bash pnpm
+pnpm add @teamhanko/passkeys-sdk
+```
+
+</CodeGroup>
+
+## Usage
+
+A `tenant` is an API client instance for one tenant of the Hanko Passkey API.
+
+[👉 See multitenancy](/passkey-api/faq#tenant-id-and-multitenancy)
+
+Create a new tenant instance:
+
+```ts
+const passkeyApi = tenant({
+	tenantId: "<your tenant id>",
+	apiKey: "<your secret api key>",
+});
+```
+
+-   If you only use public API methods, like `/login/initialize`, you can omit the `apiKey`.
+-   If you're self-hosting the Passkey API, make sure to pass the `baseUrl` as well.
+
```

Comment:
Maybe add a warning that the `apiKey` should be protected and kept secret. Since you can use this SDK on frontend and backend and using the `apiKey` on the frontend is a security issue.